tests/mpath_test.py

```diff
@@ -49,6 +49,22 @@ def test_is_mpath_member(self):
         # device and no error is reported
         self.assertFalse(BlockDev.mpath_is_mpath_member("/dev/loop0"))
 
+    @tag_test(TestTags.NOSTORAGE)
+    def test_get_mpath_members(self):
+        """Verify that get_mpath_members works as expected"""
+        ret = BlockDev.mpath_get_mpath_members()
+        self.assertIsNotNone(ret)
+
+    @tag_test(TestTags.NOSTORAGE)
+    def test_set_friendly_names(self):
+        """Verify that set_friendly_names works as expected"""
+        flag=os.system('ls /etc/multipath.conf')
```

Comment:
Please use `shutil.which` here to check for `mpathconf`. If you need to check for the file existence then use `os.path.exists`. Thanks.

Reply:
ok，It has been modified